index.js

```diff
@@ -1,10 +1,10 @@
 const fromEvent = (node, name) => (start, sink) => {
   if (start !== 0) return;
   const handler = ev => sink(1, ev);
+  node.addEventListener(name, handler);
   sink(0, t => {
     if (t === 2) node.removeEventListener(name, handler);
   });
-  node.addEventListener(name, handler);
```

Comment:
If the event source sends synchronous emissions to the event listener, then this implementation would end up calling `sink(1,ev)` before it called `sink(0, ...)`, which doesn't satisfy the Callbag spec. Instead I suggest you revert the ordering how it was before, but to fix this issue, invert a boolean flag in `if (t === 2)` so that before adding the event listener, we check if the boolean was not inverted.